scripts/dev.sh

```diff
@@ -84,24 +82,25 @@ start-hardhat_node() {
   # Account #19: 0x5a485c203d9537095a6be2acc5a7ad83805d301d (10000 ETH)
   # Private Key: 0xb86f3287c11a77c7317c2484be2bd386816876ead8ceaf86971b7b7c1afbb12b
 
-  ganache_pid=$!
+  hardhat_pid=$!
 
-  echo "Waiting for ganache to launch..."
+  echo "Waiting for hardhat to launch..."
 
-  while ! ganache_running; do
+  while ! hardhat_running; do
     sleep 0.1 # wait for 1/10 of the second before check again
   done
 
   echo "Harhat node launched!"
 }
 
-if ganache_running; then
-  echo "Using existing hardhat node instance"
-else
-  echo "Starting our own hardhat node instance"
-  start-hardhat_node
+if hardhat_running; then
+  echo "Killing existent hardhat"
+  fuser -k 8545/tcp
```

Comment:
Fuser is only Linux (I believe?). 
Something like this should work for both Linux/macOS
```
kill $(lsof -t -i:8545) 
```

Reply:
done in https://github.com/DXgovernance/dxvote/pull/687/commits/b0cca80c4b9fc3a521958523e6e6bcc46749023d